sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -198,6 +198,18 @@ if("native_cpu" IN_LIST SYCL_ENABLE_PLUGINS)
       OpenCL-Headers
   )
 
+  if(NOT DEFINED NATIVECPU_USE_OCK)
+    option(Native_CPU_USE_OCK "Use the oneAPI Construction Kit for Native CPU" On)
+  endif()
+
+  if(Native_CPU_USE_OCK)
+    message("Compiling Native CPU adapter with OCK support.")
+    target_compile_definitions(ur_adapter_native_cpu PRIVATE NATIVECPU_USE_OCK)
+  else()
+    message("Compiling Native CPU adapter without OCK support.
```

Comment:
[message()](https://cmake.org/cmake/help/latest/command/message.html) command expects a _mode_ as the first argument. In this case I believe `WARNING` would be appropriate.

Reply:
Yes you are right, I've added `STATUS` and `WARNING` where appropriate, thank you